features/org.wso2.carbon.appmgt.store.feature/src/main/resources/admin-dashboard/site/themes/default/templates/businessowners-task/js/businessowners.js

```diff
@@ -276,30 +276,29 @@ $(document).on("click", ".owner-edit-button", function () {
 $(document).on("click", ".owner-delete-button", function () {
     var ownerId = $(this).data("ownerId");
     var ownerName = $(this).closest('tr').find("td:first").text();
-    $.ajax({
-               url: context + '/apis/businessowners/' + ownerId,
-               type: 'DELETE',
-               contentType: 'application/json',
-               dataType: 'json',
-               success: function (response) {
-                   if(response) {
-                       $(".alert").alert();
-                       var isConfirmed = confirm("Are you sure you want to delete the business owner " + ownerName + "?");
-                       if (isConfirmed) {
+    var isConfirmed = confirm("Are you sure you want to delete the business owner " + ownerName + "?");
+    if (isConfirmed) {
+        $.ajax({
+                   url: context + '/apis/businessowners/' + ownerId,
+                   type: 'DELETE',
+                   contentType: 'application/json',
+                   dataType: 'json',
+                   success: function (response) {
+                       if(response) {
```

Comment:
Formatting issues are fixed in https://github.com/wso2/carbon-appmgt/pull/539/files